### DISCUSSION TITLE
Remote Denied Access to git push

### DISCUSSION THREAD

**Discussion by coffeerow-ai** [community/community#116858] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

PS C:\Users\noahi\Documents\minitorch-module-0-coffeerow-ai\workspace\venv\module-0> git push 
remote: Permission to coffeerow-ai/module-0.git denied to coffeerow.
fatal: unable to access 'https://github.com/coffeerow-ai/module-0.git/': The requested URL returned error: 403 

Need help, please! 

Thank you.

**Comment by Sharon-SHH** [comment]
This issue may be attributed to either your GitHub settings on your computer or an expired GitHub token

**Reply by coffeerow-ai** [reply]
Thank you for responding- by any chance can you elaborate on the github setting or expired github token?

**Reply by Sharon-SHH** [reply]
token: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens
Git on computer setting: https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration
